### PR TITLE
[SYNPYTH-91] add reset_link support for webhooks

### PR DESCRIPTION
**Pull request by Zhebr**

### CODE REVIEW

File: syncano/models/base.py [hunk 1]
```diff
@@ -1027,6 +1031,18 @@ def run(self, **payload):
         response.update({'instance_name': self.instance_name, 'webhook_name': self.name})
         return WebhookTrace(**response)
 
+    def reset(self, **payload):
+        """
+        Usage::
+
+            >>> wh = Webhook.please.get('instance-name', 'webhook-name')
```
Comment (dancio) [inline syncano/models/base.py]: wrong doc string

**Comment by dancio** [comment]
lgtm

File: syncano/models/base.py (hunk 1 above)
Comment (Zhebr) [inline syncano/models/base.py]: what is wrong here?

Comment (dancio) [inline syncano/models/base.py]: my bad it's ok

**Pull request merged**